packages/ember-routing/tests/system/dsl_test.js

```diff
@@ -1,13 +1,25 @@
 import isEnabled from 'ember-metal/features';
+import Ember from 'ember-metal/core';
 import EmberRouter from 'ember-routing/system/router';
 
-var Router;
+var Router, oldWarn;
+
+function confirmWarns(expectedMsg) {
```

Comment:
There is no general helper for `expectWarn`? I would not be surprised, but we do have them for asserts and deprecations so it is worth checking.